### DISCUSSION TITLE
What is the lem equivalent of Emacs' which-key?

### DISCUSSION THREAD

**Discussion by g-gundam** [lem-project/lem#1505] [Q&A]
https://github.com/justbur/emacs-which-key

>`which-key` is a minor mode for Emacs that displays the key bindings following your currently entered incomplete command (a prefix) in a popup. For example, after enabling the minor mode if you enter C-x and wait for the default of 1 second the minibuffer will expand with all of the available key bindings that follow C-x (or as many as space allows given your settings). 

![example](https://github.com/justbur/emacs-which-key/raw/master/img/which-key-bottom.png)

----

Does lem have something like which-key?

**Comment by vindarel** [comment]
Not yet. In my wish list too.

Meanwhile, you have: https://lem-project.github.io/usage/usage/#describe-keys

> To see the available key bindings in the current keymap, use M-x describe-bindings.

> To know what function is bound to a key binding, use C-x ? (M-x describe-key).

> You can do more and list all Lem’s existing keybindings with M-x documentation-describe-bindings. It inspects Lem and lists keybindings, grouped by modes.

**Reply by g-gundam** [reply]
I was looking for `describe-key`.  I can't believe I missed it.  I was using `describe-bindings` before.
